src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getCommentsFromPostsById(postsId) {
```

Comment:
This function finds Comments, so maybe the appropriate name will be findCommentsFromPostsById

Reply:
I get an array of comments instead of just one comment